WooCommerce/Classes/ViewRelated/ReusableViews/SwiftUI Components/EmptyState.swift

```diff
@@ -35,6 +46,7 @@ private extension EmptyState {
         static let verticalSpacing: CGFloat = 16
         static let horizontalSpacing: CGFloat = 24
         static let width: CGFloat = 168
+        static let buttonWidth: CGFloat = 228
```

Comment:
This width is matching the button width on `EmptyStateViewController`:

https://github.com/woocommerce/woocommerce-ios/blob/398e90e0ffe7f9b86bf7dcf5173c1c41220d4e0b/WooCommerce/Classes/ViewRelated/ReusableViews/EmptyStateViewController/EmptyStateViewController.xib#L57